daprdocs/content/en/getting-started/install-dapr-selfhost.md

```diff
@@ -125,4 +125,4 @@ explorer "%USERPROFILE%\.dapr\"
 
 <br>
 
-{{< button text="Next step: Try Dapr quickstarts >>" page="getting-started/_index.md" >}}
\ No newline at end of file
+{{< button text="Next step: Try Dapr quickstarts >>" page="getting-started/quickstarts" >}}
```

Comment:
@yauri-io  This page should link to the next page in the TOC which is the "Use the Dapr API". can you please change the page to redirect to:

page="getting-started/get-started-api.md"